### PR TITLE
85 investigate and resolve mismatch in catch and trap data

### PR DESCRIPTION
**Pull request by alvizek**
Battle and Clear have mismatches because of interpolated catch data. Knights had mismatches because we were using trap data from spreadsheets and catch data from CAMP. Knights uses CAMP 2006-present but collected data before 2006. I pulled in trap data from CAMP and also merged CAMP with pre2006 data to resolve mismatches. Please check through and then merge!

### CODE REVIEW

File: data-raw/standard-format-data-prep/rst_catch_standard_format.Rmd [hunk 1]
```diff
@@ -137,7 +133,7 @@ lifestage_encoding <- function(dat) {
                                  lifestage %in% c("fry", "button-up fry", "fry (button-up)") ~ "fry",
                                  lifestage == "smolt" ~ "smolt",
                                  lifestage == "adult" ~ "adult",
-                                 lifestage == "yearling" ~ "yearling"))
+                                 lifestage %in% c("yearling", "older juvenile") ~ "yearling"))
```
Comment (ErinCain) [inline data-raw/standard-format-data-prep/rst_catch_standard_format.Rmd:136]: Could older Juvenile also be a smolt/silvery parr? 
Did you look at the fork length, timing, and total amount that are categorized this way?

File: data-raw/standard-format-data-prep/rst_catch_standard_format.Rmd [hunk 2]
```diff
@@ -519,6 +528,28 @@ knights_rst_clean <- knights_landing_rst %>%
...
+  select(-c(start_date, stop_date, location, fork_length_max_mm, fork_length_min_mm, cpue)) %>% 
+  rename(run = at_capture_run,
+         adipose_clipped = marked) %>% 
+  mutate(adipose_clipped = case_when(adipose_clipped == T ~ "Natural",
+                                     adipose_clipped == F ~ "Hatchery",
+                                     T ~ "Unknown"),
+         count = ifelse(is.na(count), 0, count),
+         # this likely happened from reshaping of data
+         remove = case_when(is.na(run) & species != "Steelhead" ~ "remove",
+                            species == "Steelhead" ~ "remove",
```
Comment (ErinCain) [inline data-raw/standard-format-data-prep/rst_catch_standard_format.Rmd:543]: Are we sure we ant to remove chinooks with no run? And steelhead? I know we went back and forth on this but I also am now thinking if we want to make this data as useful as possible to a wider audience we might as well leave in other species

**Review by ErinCain — COMMENTED** [review]
Looks good. I had a couple of questions before merging

File: data-raw/standard-format-data-prep/rst_catch_standard_format.Rmd (hunk 1 above)
Comment (alvizek) [reply data-raw/standard-format-data-prep/rst_catch_standard_format.Rmd:140]: Only Knights has older juvenile. They used this in their spreadsheets, not CAMP. We are only using spreadsheets pre2006 and there are no older juveniles during this time. I added a note to explain this.

File: data-raw/standard-format-data-prep/rst_catch_standard_format.Rmd (hunk 2 above)
Comment (alvizek) [reply data-raw/standard-format-data-prep/rst_catch_standard_format.Rmd]: I added steelhead back in and also reran CAMP data for Knights to include all species. I will do the same for Feather and maybe Yuba if they have other species.

Comment (ErinCain) [reply data-raw/standard-format-data-prep/rst_catch_standard_format.Rmd]: Thank you!

**Review by ErinCain — APPROVED** [review]

**Pull request merged**